mesosdriver/executor_driver.go

```diff
@@ -174,6 +174,14 @@ func (driver *ExecutorDriver) buildEventHandler() events.Handler {
 			)
 
 		},
+
+		executor.Event_HEARTBEAT: func(_ context.Context, e *executor.Event) error {
+			// We don't process heartbeats. In theory we ought to count how many we get
+			// and force reconnect if we don't get one. But we already watch the
+			// connection so it's just redundant. Ignore.
+			log.Debug("Heartbeat received")
```

Comment:
Nit picking here. Would suggest to call this "Heartbeat ignored" or "Ignoring hearbeat" to make it clear(er) what is happening.

Reply:
It's just a received event like the rest of them in there. I'm going to keep it like this. I think the comment is explanatory enough.